src/order/jordan_hoelder.lean

```diff
@@ -0,0 +1,803 @@
+/-
+Copyright (c) 2021 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes
+-/
+import order.lattice
+import data.fin
+import data.set_like.basic
+import data.list.sort
+import data.equiv.fin
+import data.equiv.functor
+/-!
+# Jordan Hölder Theorem
+
+This file proves the Jordan Hölder theorem for a `jordan_hoelder_lattice`, a class also defined in
```

Comment:
Hölder is always "romanised" as Holder, as far as I'm aware.

Reply:
I personally quite like `hoelder`.

Reply:
Can you find any textbook that spells it this way? This is silly.

Reply:
Swapped to `holder`